storage/innobase/include/srv0srv.h

```diff
@@ -985,6 +985,48 @@ bool
 srv_is_undo_tablespace(
 	ulint	space_id);
 
+/** Page gragmentation statistics */
```

Comment:
srv source subtree and srv_ function name prefix are not really appropriate anymore given the status var moved from server singleton to THD. I'd move it to btr

Reply:
I disagree:
1. We need `fragmentation_stats_t` to be a part of `export_var_t` which is located in the `srv0srv.h`
2. We need to call `srv_get_fragmentation_stats()` from `srv_export_innodb_status()` located in `srv0srv.cc`.
3. Implementing `srv_get_fragmentation_stats()` / `srv_add_fragmentation_stats()` requires knowledge of `THD` data structure internals. Even in `srv0srv.cc` it is done via a hack with
```c++
#define MYSQL_SERVER /* For THD class */
#include "sql_class.h"
#undef MYSQL_SERVER
```
Doing the same from BTR level in my opinion is even worse.
So, I would prefer to not move these functions.